sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/CliSuite.scala

```diff
@@ -202,7 +201,10 @@ class CliSuite extends SparkFunSuite with BeforeAndAfterAll with Logging {
   }
 
   test("Commands using SerDe provided in --jars") {
-    val jarFile = HiveTestUtils.getHiveHcatalogCoreJar.getCanonicalPath
+    val jarFile = "../hive/src/test/resources/" +
```

Comment:
Adding these JARs to the source tree has some LICENSE and NOTICE implications. The Hive NOTICE text from these JARs would have to go in NOTICE (and probably NOTICE-binary as we published test JARs). They'd have to be listed in LICENSE and LICENSE-binary too.

This is possible, but is it equally possible to just download these like we do with other Hive jars?